### PR TITLE
Add "sein" to German definitions of English "be" verbs.

### PR DESCRIPTION
**Pull request by dlyongemallo**

### CODE REVIEW

**Comment by De7vID** [comment]
@Cyberman-tM @LievenLitaer Please verify if these substitutions are correct.

File: mem-14-r.xml [hunk 1]
```diff
@@ -6465,7 +6465,7 @@ This was used for "Ok" (while {qIl:v} was used for "Cancel") in Klingon Blockly.
       <column name="entry_name">run</column>
       <column name="part_of_speech">v:is</column>
       <column name="definition">be short (in stature)</column>
-      <column name="definition_de">klein, kurz (in Größe)</column>
+      <column name="definition_de">klein, kurz sein (in Größe)</column>
```
Comment (LievenLitaer) [inline mem-14-r.xml]: better: "klein sein (körperliche Größe)"
In German, people are not "short".

File: mem-15-S.xml [hunk 2]
```diff
@@ -6859,7 +6859,7 @@ This verb is also used for the encryption of data.[3]</column>
       <column name="entry_name">Sub</column>
       <column name="part_of_speech">v:1,is</column>
       <column name="definition">be solid</column>
-      <column name="definition_de">fest, hart, solide</column>
+      <column name="definition_de">fest, hart, solide sein</column>
```
Comment (LievenLitaer) [inline mem-15-S.xml]: better: "fest, solide sein"
(not every solid is hard)

File: mem-14-r.xml [hunk 3]
```diff
@@ -3592,7 +3592,7 @@ This is the same word as the verb {ret:v}.[2]</column>
       <column name="entry_name">rIb</column>
       <column name="part_of_speech">v:is</column>
       <column name="definition">be spread, spread out</column>
-      <column name="definition_de">verteilen, auffächern</column>
+      <column name="definition_de">verteilen, auffächern sein</column>
```
Comment (LievenLitaer) [inline mem-14-r.xml]: verstreut, verteilt sein

File: mem-09-n.xml [hunk 4]
```diff
@@ -5323,7 +5323,7 @@ See {SIq:v} for an explanation of the usage of these finger verbs.</column>
       <column name="entry_name">nIv</column>
       <column name="part_of_speech">v:is</column>
       <column name="definition">be superior</column>
-      <column name="definition_de">überlegen, höher, besser</column>
+      <column name="definition_de">überlegen, höher, besser sein</column>
```
Comment (LievenLitaer) [inline mem-09-n.xml]: überlegen, höhergestellt sein

File: mem-06-j.xml [hunk 5]
```diff
@@ -1330,7 +1330,7 @@ Note that in Klingon, "deep space" is called {logh Hop:n}, literally, "far space
       <column name="entry_name">jaQHa'</column>
       <column name="part_of_speech">v:is,deriv</column>
       <column name="definition">be shallow</column>
-      <column name="definition_de">seicht, untief</column>
+      <column name="definition_de">seicht, untief sein</column>
```
Comment (LievenLitaer) [inline mem-06-j.xml]: flach, seicht, nicht tief sein

File: mem-05-H.xml [hunk 6]
```diff
@@ -4388,7 +4388,7 @@ Watch Gowron say this: {YouTube video:url:http://youtu.be/KArQlviVXMk}</column>
       <column name="entry_name">HIr</column>
       <column name="part_of_speech">v:is</column>
       <column name="definition">be stuck (physically fastened to something)</column>
-      <column name="definition_de">abgestempelt</column>
+      <column name="definition_de">abgestempelt sein</column>
```
Comment (LievenLitaer) [inline mem-05-H.xml]: festsitzen, festkleben
[as a side note: "be stuck" is not an adjective in German, in neither form]

File: mem-15-S.xml (hunk 2 above)
Comment (De7vID) [reply mem-15-S.xml]: I'm not even sure that this adjectival sense of "be solid" has anything to do with solids (meaning the state of matter). To "be solid" could simply mean to be firm or reliable (in character), or not hollowed out (of a physical object). The slang meaning seems to suggest that it has to do with character. But in the absence of more information I guess this is fine.

**Comment by De7vID** [comment]
After this change, these are the German definitions for which the English definition starts with "be ...". These were skipped because I could not programmatically determine whether a German word ending in "-en" was an infinitive or an adjective. I suppose I could've been slightly more clever and filtered for "ge-...-en", like "geschlossen", "gemahlen", and "gebogen", which I think need "sein".

```xml
mem-01-b.xml-          <column name="definition_de">sich komplett irren</column>
mem-01-b.xml-      <column name="definition_de">geboren werden</column>
mem-01-b.xml-      <column name="definition_de">handeln um (etwas)</column>
mem-02-ch.xml-      <column name="definition_de">sich nicht wohlfühlen</column>
mem-02-ch.xml-      <column name="definition_de">sich wohlfühlen</column>
mem-02-ch.xml-      <column name="definition_de">übrig bleiben</column>
mem-03-D.xml-      <column name="definition_de">klemmen, feststecken, sich nicht mehr bewegen</column>
mem-03-D.xml-      <column name="definition_de">Glück haben</column>
mem-03-D.xml-      <column name="definition_de">kein Glück haben</column>
mem-04-gh.xml-      <column name="definition_de">er benimmt sich idiotisch</column>
mem-05-H.xml-      <column name="definition_de">festsitzen, festkleben</column>
mem-06-j.xml-      <column name="definition_de">kooperieren, zusammenarbeiten</column>
mem-07-l.xml-      <column name="definition_de">fest stecken</column>
mem-07-l.xml-      <column name="definition_de">in einer bestimmten Pose stehen (Personen, Tiere oder Kampfsport)</column>
mem-07-l.xml-      <column name="definition_de">in einer Fluglage befinden (Flugzeug)</column>
mem-07-l.xml-      <column name="definition_de">von etwas umgeben</column>
mem-09-n.xml-      <column name="definition_de">seitlich, von der Seite, seitwärts bewegen</column>
mem-10-ng.xml-      <column name="definition_de">stabil, ausgeglichen</column>
mem-10-ng.xml-      <column name="definition_de">feststecken</column>
mem-11-p.xml-      <column name="definition_de">eingestuft, eingeordnet, einen Rang haben</column>
mem-11-p.xml-      <column name="definition_de">indirekt, verschlagen</column>
mem-11-p.xml-      <column name="definition_de">gemahlen</column>
mem-11-p.xml-      <column name="definition_de">genug (von etw.) haben, es satt haben</column>
mem-12-q.xml-      <column name="definition_de">ein Problem darstellen</column>
mem-12-q.xml-      <column name="definition_de">selten</column>
mem-13-Q.xml-      <column name="definition_de">bedauern, bereuen</column>
mem-14-r.xml-      <column name="definition_de">mit einander verbunden</column>
mem-14-r.xml-      <column name="definition_de">(am Steg) festgemacht, vor Anker liegen</column>
mem-14-r.xml-      <column name="definition_de">fertig, abgeschlossen</column>
mem-14-r.xml-      <column name="definition_de">kurz davor zu zerfallen</column>
mem-15-S.xml-      <column name="definition_de">(im Stau) feststecken</column>
mem-15-S.xml-      <column name="definition_de">geschlossen</column>
mem-15-S.xml-      <column name="definition_de">tapfer, heldenhaft, kühn, mutig, unerschrocken</column>
mem-18-v.xml-      <column name="definition_de">sich schwindlig fühlen</column>
mem-19-w.xml-      <column name="definition_de">gekrümmt, gebogen</column>
mem-19-w.xml-      <column name="definition_de">schief gelaufen</column>
mem-19-w.xml-      <column name="definition_de">gestreift, mit Linien versehen</column>
mem-19-w.xml-      <column name="definition_de">Ungnade fallen</column>
mem-20-y.xml-      <column name="definition_de">weggenommen</column>
mem-20-y.xml-      <column name="definition_de">zufrieden</column>
mem-21-a.xml-      <column name="definition_de">etw. verbrauchen</column>
mem-27-extra.xml-      <column name="definition_de">landen? starten? (Flugzeug)</column>
```

There's an inconsistency between the way `ghaH vuv SuS neH` is defined in English and German, respectively "be foolish, be a fool" and "er benimmt sich idiotisch". Arguably the German is closer to what KGT says the idiom means, whereas the English is a condensation of the meaning. I think I will change the English meaning, unless you have another suggestion for the German.

**Comment by LievenLitaer** [comment]
Indeed, some of them still need the sein addition, like "gekrümmt sein", gestreift sein and some more. But indeed, there are several english words with "be" in their definirion, which are not ncesessarily adjectives, like "be born". I'll look at the above next week whe I'm back from my trip.

**Pull request merged**